.github/workflows/changelog-verify.yml

```diff
@@ -34,9 +34,75 @@ jobs:
     runs-on: ubuntu-latest
     # require the first job to have ran
     needs: changedfiles
+    # Map a step output to a job output
+    outputs:
+      all: ${{ steps.validate.outputs.all }}
     # only run if there are changed files
     if: ${{needs.changedfiles.outputs.all}}
     steps:
       - uses: actions/checkout@v3
       - name: Verify changelogs
-        run: bash ./.githooks/pre-receive "${{needs.changedfiles.outputs.all}}"
+        id: validate
+        # Set outputs using the command.
+        run: |
+          bash ./.githooks/pre-receive "${{needs.changedfiles.outputs.all}}" 2>&1 | tee output.txt
+          echo "all=$(cat output.txt | jq -sR .)" >> $GITHUB_OUTPUT
+  review:
+    runs-on: ubuntu-latest
+    # require the validate job to have ran
+    needs: validate
+    if: always()
+    steps:
+      - name: Verify changelogs KO
+        if: ${{ contains(needs.validate.outputs.all, 'failed') }}
+        uses: actions/github-script@v5
+        with:
+          script: |
+            const result = `<!-- validate_review -->The changelog validation failed with the following output:
```

Comment:
Should we use `validate_review_chaenlogs` as the unique identifier? 

I can see us following this pattern for other times of comments as well

Reply:
Side note: I'm on the waitlist for https://reviewpad.com/openai-waitlist

Reply:
Cool! Re: ChatGPT... So far my experience hasn't been great, I had a couple of hiccups with this PR and I asked for help... the results made sense but just didn't work. I'll keep trying, if anything, it could be inspiration

Reply:
Keep trying. It's just a new skill we have to learn.